app/assets/stylesheets/application.scss

```diff
@@ -23,6 +23,10 @@ a:visited {
   color: $govuk-blue;
 }
 
+a[data-method='delete'] {
```

Comment:
We had a conversation about this convention. We don't use colour to infer action in gov.